jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/colibri/ColibriV2SessionManager.kt

```diff
@@ -371,6 +395,10 @@ class ColibriV2SessionManager(
                         // The conference on the bridge has expired. The state between jicofo and the bridge is out of
                         // sync.
                         throw ColibriAllocationFailedException("Conference not found", true)
+                    } else if (reason == Colibri2Error.Reason.CONFERENCE_ALREADY_EXISTS) {
```

Comment:
I don't think we'd see `condition = item_not_found` and `reason = CONFERENCE_ALREADY_EXISTS`. When a colibri conference exists the bridge should respond with `condition = conflict` (and `reason = CONFERENCE_ALREADY_EXISTS`)

Reply:
You are right about it, sorry for that. Moved it to the proper place.